deps/cuda1.jl

```diff
@@ -3,9 +3,13 @@
 fp = open("cuda1.cu","w")
 #using Knet: unary_ops
 
-function cuda1src(f, j=f, ex="$f(xi)"; BLK=256, THR=256)
+include("gamma.jl")
+print(fp,cuda1gammafamily())
+
+function cuda1src(f, j=f, ex="$f(xi)"; seperate_impl=false, BLK=256, THR=256)
```

Comment:
@denizyuret I amended the `cuda1src` function to support functions which have different CUDA kernel implementation for float and double. Before the kernel is assumed to be same. Does it look good?

Reply:
@CarloLucibello This float v.s. double issues is solved by this.